### DISCUSSION TITLE
Friction 0.9.1

### DISCUSSION THREAD

**Discussion by rodlie** [friction2d/friction#29] [Announcements]
Introducing Friction, a fast and flexible 2D motion graphics application for Linux and Windows.

* Export to SVG animation
* Export to any video/image format supported by ffmpeg
* JS expressions
* Support for vector and raster effects

Friction is a continuation/fork of enve with the goal of making a production-ready application for web animations/videos.

**This application is currently beta quality, expect missing and/or broken features. 0.9.x releases may also break compatibility and/or change UI at any time until we reach version 1.0.**

![Screenshot](https://friction.graphics/assets/friction-screenshot-01.jpg)
![Screenshot](https://friction.graphics/assets/friction-screenshot-02.jpg)
![Screenshot](https://friction.graphics/assets/friction-screenshot-03.jpg)

### Requirements

* OpenGL 3.3 compatible graphics card and driver
* Display resolution of 1920 x 1080 or better is recommended
* 16GB RAM is recommended
* Windows 10 / 11 x64
* Linux x86_64
  * Ubuntu 22.04 LTS / 23.04
  * Red Hat Enterprise Linux 9.1+
  * Fedora 38

### Notes

* Windows builds are not signed
* Fedora builds are untested
* Sources for third-party components can be downloaded from [SourceForge](https://sourceforge.net/projects/friction/files/source)
* Build instructions can be found [here](https://github.com/friction2d/friction/blob/main/docs/Linux-build.md)

### Known Issues

* No documentation *(not a priority, contributions are welcome)*
* CSS style sheets are not supported when importing an SVG #28 
  * SVG's from Adobe Illustrator (and others?) may have issues due to this 

### Checksums *(sha256)*

* `3c1accc8f516c60d6f98842d7258d4e663328366f19ae6b6f96156c715030d8b` `friction-0.9.1-setup-win64.exe`
* `9dec4f87dc4aa2f4804f67247e961de82074c8f580f90a72c837239d842a7de4` `friction-0.9.1-win64.7z`
* `aef4ed863ab472951df08dc8f8169c1b683d6bc4ab57e1d908e064ef6627dafd` `friction-0.9.1-ubuntu-23.04.deb`
* `b2487ae05ad53eeff0d5cdbec8e503d27bb43cd1c7a899f14b53ba4e0ca500b2` `friction-0.9.1-ubuntu-22.04.deb`
* `7727e2955a95e1660fe4c48c64d424651e16244d345c70df4ff80c2bad1a7eea` `friction-0.9.1-el9.rpm`
* `4606799a2d711b0b3c5b0cea1287dfd01161781caed5efb8cbc526c74ad422f2` `friction-0.9.1-fc38.rpm`

<hr /><em>This discussion was created from the release <a href='https://github.com/friction2d/friction/releases/tag/v0.9.1'>Friction 0.9.1</a>.</em>

**Comment by sanjuchopracool** [comment]
Can you also add instruction to build on windows ?

**Reply by rodlie** [reply]
Sure, will do it later today.

**Comment by dewinmaya** [comment]
Fill Over Stroke: Currently, when you increase the thickness of the stroke, it overlaps the fill, it would be nice to have the option to select if you want the stroke over the fill or the fill over the stroke. Excuse me, use a translator, I don't speak English.
